lte/gateway/c/core/oai/tasks/amf/nas_proc.cpp

```diff
@@ -645,6 +644,9 @@ int amf_decrypt_imsi_info_answer(itti_amf_decrypted_imsi_info_ans_t* aia) {
                                (supi_imsi.plmn.mnc_digit3 & 0xf);
   }
 
+  imsi64 = amf_imsi_to_imsi64(params->imsi);
```

Comment:
Since imsi64 is obtained here, it can be removed from line number 656.

Reply:
At line number, https://github.com/magma/magma/pull/12088/files#diff-359e73baac363f6d67312a481dde16bd3d45d72538d19589bb94f57e5ac29385R593 imsi64 is already fetched; 
Can we update that value to ue-context?

Reply:
addressed